onnxruntime/core/providers/dml/DmlExecutionProvider/src/Operators/GeneratedShaders/grid_sample_int_fp16.h

```diff
@@ -0,0 +1,6291 @@
+#if 0
```

Comment:
Why all the headers files begin with "#if 0" ?

Reply:
They are generated by dxc/fxc, not sure why they begin with the if deffed section.